### DISCUSSION TITLE
Is it possible to use multiple apps on one SQLite database?

### DISCUSSION THREAD

**Discussion by Majoramari** [prisma/prisma#10733] [Q&A] [answered]
I'm connecting to one Sqlite database from 3 apps

I'm getting this error:
```
PrismaClientUnknownRequestError:
Invalid `cData.update()` invocation in
C:\Users\Major\Documents\github\Rayila\src\Modules\Utils.ts:62:45

   59
   60 // * update
   61 if (data && voice_channel_id) {
→  62     const updatedData = await cData.update(
  Error occurred during query execution:
ConnectorError(ConnectorError { user_facing_error: None, kind: ConnectionError(Timed out during query execution.) })
    at cb (C:\Users\Major\Documents\github\Rayila\node_modules\@prisma\client\runtime\index.js:38687:17) {
  clientVersion: '3.6.0'
}
```

at first I thought It may happen when 2 apps try to reach the database at once but I test that and it's not the case, I get this error randomly so far..

**Comment by buraks** [comment]
I get the same error. Adding `?connection_limit=1` at the end of the connection string helped with these errors in my case.

However, now I get occasional `Error in connector: Error querying the database: disk I/O error` now.

I wonder if this is SQLite's limitation, some say enabling WAL allows multiple connections on SQLite to work better. I am struggling to find out how _or even if_ WAL can be enabled while using Prisma.

**Reply by janpio** [reply, marked as answer]
Right now not supported, but tracked in https://github.com/prisma/prisma/issues/3303

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
